Src/WitsmlExplorer.Frontend/__testUtils__/testUtils.tsx

```diff
@@ -194,10 +195,22 @@ export function getJobInfo(overrides?: Partial<JobInfo>): JobInfo {
     report: null,
     progress: 0,
     isCancelable: false,
+    linkType: "",
     ...overrides
   };
 }
 
+export function getReport(): BaseReport {
```

Comment:
Add overrides like the other methods in this file to make it more usable.

Reply:
done